### DISCUSSION TITLE
Feature Request: Tap status bar to scroll to *bottom*

### DISCUSSION THREAD

**Discussion by rjonesy** [community/community#18] [Mobile]
We all know tapping status bar scrolls to top. But when I am already scrolled to top, I would like a scroll bar tap to scroll to **bottom**!

Why? Because the most recent comment is always all the way at the bottom, so we have to flick upward to get to the bottom each time. 

You could also try double-tap of status bar.

**Comment by rnystrom** [comment]
You can long-press on the down arrow on a card and it'll scroll to the bottom. The gesture is a bit hidden, and this card design is likely going to change soon. If you start using this and it becomes a big part of how you navigate issues and pull requests let me know! That would help us decide if we should keep these up/down arrows.

![IMG_FC6B8FB4672F-1](https://user-images.githubusercontent.com/739696/96913589-d3ab2200-1471-11eb-9fcb-de9b5b75436c.jpeg)

**Comment by rjonesy** [comment]
OMG, can't believe I missed that because I'm 99% sure I celebrated their usefulness in prior feedback.🤦‍♂️ That said, I know exactly why I missed it (and you probably do too): That bar is the "comment draw" in my brain... it doesn't affect the upper screen.

_For my use case_ you should optimize for fast-as-hell Comments, Assignees, Labels, etc – doing whatever you have to do to improve that, and let the cards fall where they may with these up/down buttons. (And add my original suggestion in this Issue 😃.)
